application/app/Http/Controllers/Admin/AdminUserController.php

```diff
@@ -0,0 +1,167 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use App\Models\AdminUser;
+use Illuminate\Validation\Rule;
+use Illuminate\Support\Facades\Hash;
+
+class AdminUserController extends Controller
+{
+    public function index(Request $request)
+    {
+        //認可ポリシー追加　ログインユーザーがオーナーか判定
+        $this->authorize('index', AdminUser::class);
+
+        $adminUsers = AdminUser::query();
+
+        //リクエスト名前　判定　取得
+        if ($request->name !== null) {
+            //AdminUserスコープ追加
+            $adminUsers->partialName($request->name);
+        }
+
+        //リクエストメール　判定　取得
+        if ($request->email !== null) {
+            //AdminUserスコープ追加
+            $adminUsers->forwardEmail($request->email);
+        }
+
+        //リクエスト権限　判定　取得
+        if ($request->authority !== null) {
+            if ($request->authority === 'owner') {
+                $adminUsers->where('is_owner', true);
+            } else if ($request->authority === 'general') {
+                $adminUsers->where('is_owner', false);
+            }
+        }
+
+        //リクエスト　並び替え方向
+        switch ($request->sort_direction) {
+            case 'asc':
+                $direction = 'asc';
+                break;
+            case 'desc':
+                $direction = 'desc';
+                break;
+            default:
+                $direction = 'asc';
+        }
+
+        //リクエスト並び替え　判定
+        switch($request->sort_column) {
```

Comment:
```suggestion
        switch ($request->sort_column) {
```
ここだけくっついてるの不自然です。
細かい部分にも気を配るようにしてください。

Reply:
並び替え方向と並び替えもスコープにしてほしいなあ。

Reply:
承知しました。

Reply:
<img width="663" alt="スクリーンショット 2020-03-03 23 50 39" src="https://user-images.githubusercontent.com/11298634/75787471-22546080-5daa-11ea-8a30-68682ad2635c.png">
フォーマット程度ならこれで簡単に直せます。

Reply:
なるほどありがとうございます。